CONTRIBUTING.md

```diff
@@ -1,17 +1,195 @@
-# Contributing to Forgerock's Javascript SDK
+# Contribution Guidelines for Ping SDK for Javascript
```

Comment:
In docs I try to always use "JavaScript" as the casing, could we do a quick search and replace?